js/interface.js

```diff
@@ -2226,7 +2368,20 @@ $('[data-save-rule]').click(function(event) {
     });
 
     rule.allow = { user: user };
-  } else {
+  } else if ($allow.data('allow') === 'tokens') {
+    selectedTokenId  = Number($('.tokens-list :selected').val());
+    var tokenFullName = _.find(integrationTokenList, function(token) {
```

Comment:
It appears ESLint setup is not working for you.

```suggestion

    var tokenFullName = _.find(integrationTokenList, function(token) {
```

Reply:
Did the changes as per your suggestion. this eslint issue not capture by ESLint plugin.